client/src/views/Category/index.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Container } from 'semantic-ui-react';
+import Header from '../../components/Header';
+import CategoryScrollBar from '../../components/CategoryScrollBar';
+
+const CategoryPage = (props) => {
+  const { match, location } = props;
+  return (
+    <>
+      <Header />
+      <CategoryScrollBar />
+      <Container style={{ paddingTop: '80px' }}>
+        <h2>Category: {match.params.name || 'undefined'}</h2>
```

Comment:
why display undefined? if the type is string then '' is the falsy value. so its better to show '' instead of undefined